apps/src/sites/studio/pages/home/_teacher_homepage.js

```diff
@@ -12,6 +12,9 @@ function showTeacherHomepage() {
   const sectionsData = document.querySelector('script[data-sections]');
   const configSections = JSON.parse(sectionsData.dataset.sections);
 
+  const script = document.querySelector('script[data-urlprefix]');
+  const urlPrefix = script.dataset.urlprefix;
```

Comment:
I'd prefer to have one data attribute with all data instead of two. Right now `sectionData === script`, but looking at the code one would guess these are two different scripts.